setup.py

```diff
@@ -156,3 +171,8 @@ def setup_requirements() -> Tuple[List[str], List[str], List[str]]:
         include_package_data=True,
         package_data={"": ["VERSION.txt"]},
     )
+
+# End timing
+end_time = time.perf_counter()
+total_time = end_time - start_time
+print(f"Total build time: {total_time:.2f} seconds")
```

Comment:
Nit: We only want to print when the script is run directly (not that we'd ever import `setup.py`).
```suggestion
    # End timing
    end_time = time.perf_counter()
    total_time = end_time - start_time
    print(f"Total build time: {total_time:.2f} seconds")
```